.github/workflows/pr.yml

```
@@ -0,0 +1,27 @@
+name: PR Check
+on:
+  pull_request:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node 12
+      uses: actions/setup-node@v1
```

why not use the latest version?

```suggestion
      uses: actions/setup-node@v2
```

---

Because I copied/pasted it from somewhere :-p